### PR TITLE
Updating inserMany to include Null when value is undefined, #12

### PR DESCRIPTION
**Pull request by lakhassane**

### CODE REVIEW

File: src/Adaptor.js [hunk 1]
```diff
@@ -130,7 +131,7 @@ function handleOptions(options) {
   if (options && options.setNull === false) {
     return false;
   }
-  return (options && options.setNull) || "'undefined'";
```
Comment (taylordowns2000) [inline src/Adaptor.js:133]: Hey @lakhassane , I missed this during our conversation earlier. it _seems_ that this actually is meant to allow the user to define a value in `options.setNull` which tells our parser to replace that specific value with `NULL`. I'm going to play around with this a bit before merging—I'd like to preserve that contract.

**Review by taylordowns2000 — COMMENTED** [review]
Not merging yet, see my comment. But I'll handle from here and chat with you on Slack.

File: src/Adaptor.js (hunk 1 above)
Comment (lakhassane) [reply src/Adaptor.js:133]: @taylodowns2000  In this case we can keep `return (options && options.setNull) || "'undefined'";`

I tried playing with it to use `setNull:1001` on the state and it worked fine.

**Comment by lakhassane** [comment]
@taylordowns2000 the commit has been included here.

**Review by taylordowns2000 — APPROVED** [review]
Brilliant. Thank you.

**Pull request merged**